ocaml/xenopsd/scripts/qemu-wrapper

```diff
@@ -302,10 +302,10 @@ def main(argv):
     # set up library preload path for qemu such that it can use jemalloc
     qemu_env = os.environ
     if "LD_PRELOAD" not in qemu_env:
-       qemu_env["LD_PRELOAD"] = "/usr/lib64/libjemalloc.so.1"
+       qemu_env["LD_PRELOAD"] = "/usr/lib64/libjemalloc.so.2"
     else:
-       qemu_env["LD_PRELOAD"] = "/usr/lib64/libjemalloc.so.1:" + qemu_env["LD_PRELOAD"]
-    qemu_env["MALLOC_CONF"] = "narenas:1,tcache:false,lg_dirty_mult:22"
+       qemu_env["LD_PRELOAD"] = "/usr/lib64/libjemalloc.so.2:" + qemu_env["LD_PRELOAD"]
+    qemu_env["MALLOC_CONF"] = "narenas:1,tcache:false,decay_ms:10000"
```

Comment:
I'd leave `decay_ms` at its default value, we don't need to override it. Previously returning pages to the OS was done immediately based on a threshold, which then created a perf vs memory usage tradeoff (and could lead to some extra syscalls of returning pages back only to allocate them again).
The new method does this based on time, and doesn't have the original perf issue, so we don't need to override it (e.g. in case in the future they remove this option too, or they change the default to a better one).
 
In fact I can't find a `decay_ms` option in the manpage https://jemalloc.net/jemalloc.3.html there is only `dirty_decay_ms` and `muzzy_decay_ms`.

Reply:
I tried.
Yes. The jemalloc official doc says the default decay time is 10 seconds, see https://jemalloc.net/jemalloc.3.html#opt.dirty_decay_ms.

But if we don't config it explicitly, xapi will not start up:
```
Aug 13 02:34:41 xrtmia-11-08 xapi: [error||37 /var/lib/xcp/xapi|dispatch:session.login_with_password D:9245b84ffadf|backtrace] session.login_with_password D:01e2c1730967 failed with exception Db_exn.Read_missing_uuid("host", "", "26463138-fd12-4c5e-8fe8-5e47567968fc")
Aug 13 02:34:41 xrtmia-11-08 xapi: [error||37 /var/lib/xcp/xapi|dispatch:session.login_with_password D:9245b84ffadf|backtrace] Raised Db_exn.Read_missing_uuid("host", "", "26463138-fd12-4c5e-8fe8-5e47567968fc")
```
After I configured `decay_ms` to 10000 ms, the problem was gone.

Reply:
You can find `decay_ms` here:
https://github.com/jemalloc/jemalloc/blob/dev/INSTALL.md?plain=1#L122

Reply:
I checked again.
There is indeed no `decay_ms` option.
```
[root@xrtmia-11-10 ~]# grep "jemalloc" /var/log/daemon.log
Aug 13 05:51:50 localhost message-switch[963]: <jemalloc>: Invalid conf pair: decay_ms:10000
Aug 13 05:51:54 localhost xcp-rrdd[1120]: <jemalloc>: Invalid conf pair: decay_ms:10000
Aug 13 05:52:06 xrtmia-11-10 touch[1863]: <jemalloc>: Invalid conf pair: decay_ms:10000
Aug 13 05:52:06 xrtmia-11-10 xapi-init[1862]: <jemalloc>: Invalid conf pair: decay_ms:10000
```

Reply:
Weird. The installation always fails when I use the default config. Tested many times: 4075165, 4075389, 4075844, 4075877, 4075902

Reply:
Removed the `decay_ms` configure.
Tested three times using a new tag, no above xapi error found.